python/plugins/langgraph/setup.py

```diff
@@ -9,7 +9,7 @@
 
 setup(
     name="composio_langgraph",
-    version="0.6.3",
+    version="0.6.4",
     author="Sawradip",
     author_email="sawradip@composio.dev",
     description="Use Composio to get array of tools with LnagGraph Agent Workflows",
```

Comment:
There's a typo in the description of the LangGraph plugin: "LnagGraph" should be "LangGraph". Consider fixing this while doing the version update.